scout/constants/igv_tracks.py

```diff
@@ -11,7 +11,7 @@
 HG38ALIAS_URL = "https://igv.org/genomes/data/hg38/hg38_alias.tab"
 HG38CYTOBAND_URL = "https://igv-genepattern-org.s3.amazonaws.com/genomes/hg38/cytoBandIdeo.txt.gz"
 
-HG38GENES_URL = "https://s3.amazonaws.com/igv.org.genomes/hg38/refGene.txt.gz"
+HG38GENES_URL = "https://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/ncbiRefSeq.txt.gz"
```

Comment:
Any chance of a `.bb` for that one?

Reply:
Couldn't find any. And MT was not displayed either. I'm not testing another track, see changes, but it's slow to load